alica_engine/include/engine/RunnableObject.h

```diff
@@ -122,6 +122,9 @@ class RunnableObject
     const IAlicaTimerFactory* _timerFactory{nullptr};
 
 private:
+    void setInput(const Blackboard* parent_bb, const KeyMapping* keyMapping);
```

Comment:
This exist in RunningPlan at the moment I guess?  Dont create unused functions.  If they are going to be used in a later PR create them in that PR